src/index.html

```diff
@@ -1,15 +1,156 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+      <fieldset>
+        <legend>Personal information</legend>
+
+        <label for="surname">Surname:
+          <input
+            type="text"
+            id="surname"
+            name="surname"
+            autocomplete="off">
+        </label>
+        <br>
+        <label for="name">Name:
+          <input
+            type="text"
+            id="name"
+            name="name"
+            required
+            autocomplete="off">
```

Comment:
```suggestion
          <input
            type="text"
            id="name"
            name="name"
            required
            autocomplete="off"
          >
```
Закрывающая скобка должна быть на одном уровне с открывающей